packages/angular_devkit/build_optimizer/src/index.ts

```diff
@@ -5,6 +5,9 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+import type * as ts from 'typescript';
```

Comment:
Any benefit of using `import type` here considering also that TypeScript is a direct dependency of this package? I was under the impression that `import type` was mainly intended to be used when experiencing issues with some edge cases with Babel and transpile module.

Reply:
not particularly.  It does prevent accident value usage but not really needed here.